src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -138,6 +139,12 @@ private URL getSiteURL() throws Exception {
 		debug_log( "Checking base URL: " + baseUrl );
 		return new URL( baseUrl + getConfig().getString( "key" ) );
 	}
+	private URL getSiteURLBackup() throws Exception {

```

Comment:
Wondering if it'd be best to just add something like a `restUrlBase` to the config so users can configure this themselves?

`restUrlBase: "/wp-json/wmc/v1/"`